apps/system/utils/Makefile

```diff
@@ -127,8 +129,10 @@ CSRCS += kdbg_free.c
 endif
 
 ifeq ($(CONFIG_ENABLE_HEAPINFO_CMD),y)
+ifeq ($(CONFIG_DEBUG_MM_HEAPINFO),y)
```

Comment:
When heapinfo command is enabled, it selects debug_mm_heapinfo. Why to add double dependency.

Reply:
This happens only if we run menuconfig. But if we use the defconfig directly, then there is possibility of compile errors. Hence, added this. 

@sunghan-chang Please request heapinfo engineer to make a proper fix for this.

Reply:
@JeonginKim Please check this.

Reply:
@kishore-sn I checked this. Please revert this change.
Anyone should NOT modify defconfig directly. That's why we leave a comment on top of defconfig as shown below:
```
#
# Automatically generated file; DO NOT EDIT.
# TinyAra Configuration
#
```
It is impossible to avoid an issue from direct modification.
So, we don't need to concern that case. Because of this change, currently `esp_wrover_kit/hello_with_tash` has a compilation error.

Reply:
@sunghan-chang 
I reverted it.